test/python/test_initializer.py

```diff
@@ -27,7 +27,6 @@
 from qiskit.tools.qi.qi import state_fidelity
 from .common import QiskitTestCase
 
-
```

Comment:
Can you reintroduce this newline? It is the only thing [[the linter is complaining about](https://travis-ci.org/QISKit/qiskit-sdk-py/jobs/325310217):
```
test/python/test_initializer.py:30:1: E302 expected 2 blank lines, found 1
```